### ISSUE TITLE
Problem downloading V1.0.0-apha.3

### ISSUE THREAD

**Issue by MSFTkiLRoy** [tylertravisty/rum-goggles#35]
You are providing huge service to the RUMBLE community with rum-goggles and it has become a staple for my stream. I want to utilize your newest v1.0.0-alpha3, however it is getting flagged/blocked by windows defender as having a trojan present. I had no with the alpha.2, however it does not connect to the RUMBLE chat any longer.

**Comment by tylertravisty** [comment]
Yeah, alpha.2 no longer works since rumble updated the website. Unfortunately, I'm not able to do anything about the anti-virus alerts. Many anti-virus vendors flag the programming language and framework I'm using as a virus. I'm not quite sure why it didn't flag alpha.2 as well.

**Comment by MSFTkiLRoy** [comment]
Same thing was happening when i was trying to download alpha.5, but it just worked without flagging. Not sure what was going on. 

either way, The v1.0.0 is working great! thank you for the response!
